assets/js/simply-rets-client.js

```diff
@@ -801,7 +803,7 @@ SimplyRETSMap.prototype.initEventListeners = function() {
             points = params.points,
             query  = params.query;
 
-        that.sendRequest(points, query).done(function(data) {
+        that.sendRequest(points, query, "reset").done(function(data) {
```

Comment:
This fixes an unrelated bug where the pagination offset isn't reset when making a new search in `[sr_map_search]`